net-im/skypeforlinux/skypeforlinux-9999.ebuild

```diff
@@ -93,8 +91,9 @@ src_install() {
 #	insinto /etc/dbus-1/system.d
 #	doins ${PN}.conf
 
-	dodoc usr/share/doc/skypeforlinux/* usr/share/skypeforlinux/*.html
-	dodoc usr/share/skypeforlinux/*.txt usr/share/skypeforlinux/LICENSE
+	dodoc usr/share/doc/skypeforlinux/*
+	dodoc usr/share/skypeforlinux/LICENSE
```

Comment:
We do not install license files in /usr/share/doc. We keep them available via LICENSE.